drivers/pwm/Kconfig.mcux

```diff
@@ -2,10 +2,18 @@
 
 # Copyright (c) 2019 Linaro Limited.
 # SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2022, NXP
 
 config PWM_MCUX
 	bool "MCUX PWM driver"
 	default y
 	depends on DT_HAS_NXP_IMX_PWM_ENABLED
 	help
 	  Enable mcux pwm driver.
+
+config PWM_WAITEN_FLAG
+	bool "PWM Wait Enable Flag"
+	depends on HAS_MCUX_PWM
```

Comment:
Rather than depending on `HAS_MCUX_PWM`, I would place this Kconfig within an `if PWM_MCU` block, to make it (and other future MCUX PWM specific Kconfigs) dependent on `PWM_MCUX`